group_vars/all

```diff
@@ -15,3 +15,21 @@ oms_prod_repo: /var/www/html/omsprod/
 oms_dev_repo: /var/www/html/omsdev/
 # OMS
 oms_git: https://github.com/MykolaKr/OMS.git
+#MySQL
+#replication: yes
+mysql_replication_master: db1
+replic_user: replic_user
+replic_userpassword: 123
+replic_db: omsdb
+# A list of the mysql DB to be created
+oms_mysql_db:                              
+  - name: omsdb
+oms_mysql_users:                              
+  - name: 056db
+    host: "%"
+    pass: 056db
+    priv: "omsdb.*:ALL"
+  - name: 055db
+    host: "%"
+    pass: 055db
+    priv: "omsdb.*:ALL"
```

Comment:
крім женкінса і дб є ще купа хостів де це непотрібне. В ол має бути тільки те і тільки те що є спільне для ВСІХ хостів.
On Apr

In group_vars/all<https://github.com/IF-061-DevOps/OMS-ansible/pull/89#discussion_r59980613>:

> +mysql_replication_master: db1
> +replic_user: replic_user
> +replic_userpassword: 123
> +replic_db: omsdb
> +# A list of the mysql DB to be created
> +oms_mysql_db:
> +  - name: omsdb
> +oms_mysql_users:
> +  - name: 056db
> +    host: "%"
> +    pass: 056db
> +    priv: "omsdb.*:ALL"
> +  - name: 055db
> +    host: "%"
> +    pass: 055db
> +    priv: "omsdb.*:ALL"


Крім того дані змінні можуть використовуватись до прикладу для створення бази дженкінса, тобто використання all можливе

-
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/IF-061-DevOps/OMS-ansible/pull/89/files/355404b0d42e34a44767b8cc94d4a96a0fea0376#r59980613>